hs/MakeElements.hs

```diff
@@ -53,6 +53,13 @@ aboutLinks = concatHtml (map stylesheet ["//netdna.bootstrapcdn.com/bootstrap/3.
                                        "static/style/graph/styles.css",
                                        "static/style/common/about.css",
                                        "static/style/common/common.css"])
+drawLinks :: H.Html
+drawLinks = concatHtml (map stylesheet ["//netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap.min.css",
+                                       "static/style/graph/styles.css",
+                                       "static/style/graph/graph_styles.css",
+                                       "static/style/common/about.css",
+                                       "static/style/common/common.css"])
```

Comment:
@Ian-Stewart-Binks mmm, good point, my mistake. I just figured there were a bunch of changes in happstack that had to be at some point updated in our personal branches.